src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const revRobot = {};
+
+  for (const key in robot) {
+    if (!revRobot[robot[key]]) {
+      revRobot[robot[key]] = key;
+    } else {
+      return null;
+    }
```

Comment:
try to avoid `if else`
```suggestion
    if (revRobot[robot[key]]) {
      return null;
    }
    
    revRobot[robot[key]] = key;
```

Reply:
and better to use `hasOwnProperty()`